memory/combat_manager.py

```diff
@@ -267,8 +284,31 @@ def _read_players(self):
                     if hex(item) == "0x0":
                         address += self.ITEM_OBJECT_OFFSET
                         continue
+                    character_definition_ptr = self.memory.follow_pointer(
+                        item, [0x68, 0x28, 0xF0, 0x0]
+                    )
+                    dead_ptr = self.memory.follow_pointer(item, [0x68, 0x28])
+                    dead = self.memory.read_bool(dead_ptr + 0xC8)
+
+                    definition_id = self.memory.read_string(
+                        character_definition_ptr + 0x11, 11
+                    )
+
+                    # Definition IDS are stored as some goofy serialized utf encoded string
+                    # We just do our best with the values that are provided to
+                    # Determine the character we are looking at
+                    match definition_id:
```

Comment:
This could be kind of wonky if `_SPOILERS` (or potential other playable characters) happen to have Z, V or G in their names.

Reply:
UTF-8 stuff I gather. I tried looking that stuff up for python, but didn't find anything helpful. Keep it simple like this until it breaks I guess.

Reply:
its even wilder than that, the utf8 encoded bitstrings 16 wide but are stored like:
`| . . . . . Z . A | . L . E . . .  . |` (`.` being `00`)